src/content/app/genome-browser/components/track-panel/trackPanelConfig.ts

```diff
@@ -24,7 +24,10 @@ export enum TrackItemColour {
 
 export type TrackItemColourKey = keyof typeof TrackItemColour;
 
-export type TrackActivityStatus = Status.SELECTED | Status.UNSELECTED;
+export type TrackActivityStatus =
+  | Status.SELECTED
+  | Status.UNSELECTED
+  | Status.PARTIALLY_SELECTED;
```

Comment:
This is no longer correct, right? We aren't storing `Status.PARTIALLY_SELECTED` in redux.

Reply:
✅